main.nf

```diff
@@ -906,6 +913,7 @@ if (!params.input_is_protein && params.protein_searcher == 'diamond'){
    */
   process translate {
     tag "${sample_sketch_id}"
+    label "process_high"
```

Comment:
process_high?

Reply:
translate as we discussed is a time taking process but does not require cpus. I would create a withlabel name of the process i.e translate and explicitly set cpus to 1 and set time to something really long. nf-predictorthologs as you mentioned before gets much bigger fastq.gz files as input to translate - so we make it parallel process such files then it might be bad?